src/components/Winner.js

```diff
@@ -0,0 +1,21 @@
+function calculateWinner(squares) {
```

Comment:
usually we won't put utils/helper function alone inside component folder.

If this come with actual component, you can put this in `utils.ts` or inside same file with component (personally okay with both)

Reply:
you can create something like 'src/lib/utils' to contain common util function as well